pkg/utilities/kubeadm/joincmd.go

```diff
@@ -19,6 +19,12 @@ const (
 	certificateKey = "certificate-key"
 )
 
+// AssetDescription provides image override properties when using cluster flavors
+type AssetDescription struct {
+	ImageRepository string `json:"iamgeRepository,omitempty"`
+	ImageTag        string `json:"iamgeTag,omitempty"`
```

Comment:
Nit: "iamgeTag" => "imageTag"

Reply:
This will be replaced when I rebase on eks-d.